frontend/src/pages/login.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+class SignUp extends Component {
```

Comment:
could we change this component name to "Login" or "Signin" since it's login page?

Reply:
This has been updated